### PR TITLE
Migrate homebrew-cask/chronograf to homebrew-core/chronograf

### PR DESCRIPTION
**Pull request by gunnaraasen**
This is a reopening of https://github.com/caskroom/homebrew-cask/pull/27693 now that a new PR to merge Chronograf into homebrew-core has been resubmitted via https://github.com/Homebrew/homebrew-core/pull/13494.

This PR should only be merged once PR https://github.com/Homebrew/homebrew-core/pull/13494 is merged.

### CODE REVIEW

File: tap_migrations.json [hunk 1]
```diff
@@ -121,5 +121,6 @@
   "yubikey-neo-manager": "caskroom/drivers",
   "yubikey-personalization-gui": "caskroom/drivers",
   "yubikey-piv-manager": "caskroom/drivers",
-  "yubiswitch": "caskroom/drivers"
+  "yubiswitch": "caskroom/drivers",
+  "chronograf": "homebrew/core"
```
Comment (commitay) [inline tap_migrations.json]: Please put this in alphabetical order.

**Comment by gunnaraasen** [comment]
Thanks for alphabetizing the list @miccal! Let me know if any other changes are required.

**Comment by gunnaraasen** [comment]
Are there any other changes required to get this merged?

**Comment by miccal** [comment]
@gunnaraasen no, but it must be merged in the core before we can remove it.

**Comment by vitorgalvao** [comment]
Confirmed as OK: https://github.com/Homebrew/homebrew-core/pull/13494#issuecomment-301278254.

**Pull request merged**